data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -153,6 +153,32 @@
     "Percent of individuals < 100% Federal Poverty Line in 2010"
 )
 
+# RSEI Aggregated Micro-data
+EPA_RSEI_NUMBER_FACILITIES_OUTPUT_FIELD = (
+    "Number of facilities affecting the tract"
+)
+EPA_RSEI_NUMBER_RELEASES_OUTPUT_FIELD = "Number of releases affecting the tract"
+EPA_RSEI_NUMBER_CHEMICALS_OUTPUT_FIELD = (
+    "Number of chemicals affecting the tract"
+)
+EPA_RSEI_AVERAGE_TOXICITY_OUTPUT_FIELD = (
+    "Average toxicity-weighted concentration of the cells in the tract"
+)
+EPA_RSEI_SCORE_OUTPUT_FIELD = (
+    "Sum of the risk-related scores of the cells in the tract"
+)
+EPA_RSEI_CSCORE_OUTPUT_FIELD = "Sum of the risk-related scores of the cells in the tract (Cancer toxicity weights)"
+EPA_RSEI_NCSCORE_OUTPUT_FIELD = "Sum of the risk-related scores of the cells in the tract (Noncancer toxicity weights)"
+EPA_RSEI_POPULATION_OUTPUT_FIELD = (
+    "Sum of the population of the cells in the tract"
+)
+
+EPA_RSEI_SCORE_PERCENTILE_RANK_FIELD = "Percentile rank for overall risk score"
```

Comment:
Please follow conventions of other fields, and don't create a new specific field for this, and instead use `field_names. EPA_RSEI_SCORE_OUTPUT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX` to reference this. Thanks!